microservices/scraper-vietgate/org.py

```diff
@@ -0,0 +1,65 @@
+def create_org(line):
+    """
+    Attempts to parse a line of data into the Org class.
+     Input:
+        line: an array of strings that could possibly contain organization data.
+        Lines are usually arranged as such [ 'name' , 'key:value' ], where key 
+        would be a specific field (one of name, phone, email, etc.) and value 
+        would be that specific piece of data for the field.
+         Some lines do not correspond to this format and do not contain useful
+        information.
+     Returns:
+        If the line contains valid organization data, then we return a serialized 
+        Org object containing the data in the input line. Otherwise, we'll return 
+        None.
+    """
+    name = phone = email = address = contact = url = ""
+    parseable = False
+    for idx, info in enumerate(line):
+        if idx == 0:
+            name = info
+            continue
+        if ":" not in info:
+            continue
+        delim = info.split(":", 1)
+        category = delim[0].lower()
+        data = delim[1].strip()
+        if category == "email":
+            email = data
+        if category == "phone":
+            parseable = True
```

Comment:
From my understanding, your setting parseable to true if it has a phone number, contact info, or address, but not for email? Would it be more effective to check that all the fields aren't null inside the Org class instead of having the boolean in the ```create_org``` function? You could even keep that logic here, but maybe not have a parseable boolean since it makes the logic slightly easier to follow.